src/main/resources/ldap_entry/searchEntry.xml

```diff
@@ -34,6 +35,7 @@
         <property expression="$func:filters" name="filters" scope="default" type="STRING"/>
         <property expression="$func:dn" name="dn" scope="default" type="STRING"/>
         <property expression="$func:attributes" name="attributes" scope="default" type="STRING"/>
+        <property expression="$func:limit" name="limit" scope="default" type="STRING"/>
```

Comment:
Can we make this as type INTEGER ?

Reply:
It is not possible to simply change the STRING to INTEGER. The usual practice is to use STRING in similar situations